python/ray/tests/test_usage_stats.py

```diff
@@ -301,6 +303,41 @@ def test_pg_stats():
         )
 
 
+def test_task_stats(reset_usage_stats):
+    @ray.remote
+    def foo():
+        pass
+
+    with ray.init(
+        _system_config={"metrics_report_interval_ms": 1000},
+    ) as ctx:
+        gcs_client = gcs_utils.GcsClient(address=ctx.address_info["gcs_address"])
+
+        wait_for_condition(
+            lambda: ray_usage_lib.get_extra_usage_tags_to_report(gcs_client).get(
+                "task_num_created"
+            )
+            == "1",
```

Comment:
Why does it start with one, is that an implementation detail / internal task? Would it be better to measure the difference to make it not dependent on the implementation details?

Reply:
according to @rickyyx, likely because driver is also counted as a task.